layouts/register-by-mail/index.html

```diff
@@ -0,0 +1,78 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    {{ partial "head.html" . }}
+  </head>
+  <body class="page-register-by-mail">
+
+    <div id="main" role="main">
+      <a class="skipnav" href="#main-content">Skip main navigation</a>
+
+      <!-- Steps BEGIN -->
+      <div class="graphic-list">
+        <img src="{{ .Site.BaseURL }}/assets/images/step-1.png">
+        <h4 class="usa-graphic-list-heading">Step 1</h4>
+        <p>Fill out your voter registration.</p>
+        <img src="{{ .Site.BaseURL }}/assets/images/step-2.png">
+        <h4 class="usa-graphic-list-heading">Step 2</h4>
+        <p>Mail it to your state.</p>
+        <img src="{{ .Site.BaseURL }}/assets/images/step-3.png">
+        <h4 class="usa-graphic-list-heading">Step 3</h4>
+        <p>Vote in the upcoming election!</p>
+      </div>
+      <!-- Steps END -->
+
+      <!-- Download form BEGIN -->
+      <b>
+        <a href="#">Download the registration form (PDF)</a>
+      </b>
+      <p>Download the form in other languages:</p>
+      <div class="width-one-half">
+        <ul class="usa-unstyled-list">
+          <li>
+            <a href="#">Bengali</a>
+          </li>
+          <li>
+            <a href="#">Chinese</a>
+          </li>
+          <li>
+            <a href="#">Hindi</a>
+          </li>
+          <li>
+            <a href="#">Japanese</a>
+          </li>
+          <li>
+            <a href="#">Khmer</a>
+          </li>
+        </ul>
+      </div>
+      <div class="width-one-half">
+        <ul class="usa-unstyled-list">
+          <li>
+            <a href="#">Korean</a>
+          </li>
+          <li>
+            <a href="#">Spanish</a>
+          </li>
+          <li>
+            <a href="#">Tagalog</a>
+          </li>
+          <li>
+            <a href="#">Vietnamese</a>
+          </li>
+        </ul>
+      </div>
+      <!-- Download form END -->
+
+      {{ partial "go-back.html" . }}
+      {{ partial "footer.html" . }}
+    </div>
+
+    {{/*
+    {{ partial "analytics.html" . }}
+    {{ partial "scripts.html" . }}
+    */}}
```

Comment:
You'll see the above partials commented out in various `layout` files since these files don't exist _yet_, Hugo needs them disabled in order to properly build / preview the site.